src/chart/parameter/component/SelectionConfirmationButton.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { Tooltip } from '@mui/material';
+import { CheckBadgeIconOutline } from '@neo4j-ndl/react/icons';
+import { IconButton } from '@neo4j-ndl/react';
+/**
+ * Returns a button to delete a property entry from the table inside the GraphChartEditModal.
+ */
+export const SelectionConfirmationButton = ({ onClick, key }) => {
+  return (
+    <Tooltip title={'Confirm'} disableInteractive key={key}>
+      <IconButton
+        key={`btn${  key}`}
+        className='logo-btn n-p-1'
+        aria-label={'btb-confirmation'}
+        size='large'
+        onClick={onClick}
+        clean
+      >
+        <CheckBadgeIconOutline className='header-icon' type='outline' />
```

Comment:
I think `<PlayIconOutline className='btn-icon-base-r' />`
is better here, at is implies an action

Reply:
Fix on "Fix review 3"